### PR TITLE
Update models.py

### PR DESCRIPTION
**Pull request by jjfallete**
Added ability to set_ignored(), assign(), and change_status() on an Alert object.

Used methods from AlertQuery implementation, modified to work with a single Alert. Necessary for instances when a query is not available, for example when an alert is selected by UID an AlertQuery object is not returned, just an Alert object.

## Pull request checklist

Please check if your PR fulfills the following requirements:

- [ ] Docs have been reviewed and added / updated if needed (for bug fixes / features)
- [ ] Tests have been added that prove the fix is effective or that the feature works.
- [ ] New and existing tests pass locally with the changes.
- [X] Code follows the style guidelines of this project (PEP8, clean code).
- [ ] Linter has passed locally and any fixes were made for failures.
- [X] A self-review of the code has been done.

## Pull request type

Please check the type of change your PR introduces:
- [ ] Bugfix
- [ X] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes (not tied to bugs/features)
- [ ] Other (please describe):

### CODE REVIEW

File: src/cbapi/response/models.py [hunk 1]
```diff
@@ -294,6 +294,21 @@ def refresh(self):
             self._info = result[0]
         self._last_refresh_time = time.time()
         return True
+    
+    def set_ignored(self, ignored_flag=True):
+        payload = {"updates": {"is_ignored": ignored_flag, "requested_status": "False Positive"}}
```
Comment (abowersox-cb) [inline src/cbapi/response/models.py:299]: If we do a set_ignored(False) here, shouldn't the "requested_status" be set to something other than "False Positive"? Just saying.

Comment (llyon-cb) [reply src/cbapi/response/models.py:299]: The possible values for `"requested_status"` are [Resolved, Unresolved, In Progress, or False Positive.](https://developer.carbonblack.com/reference/enterprise-response/6.3/rest-api/#bulk-update-alerts)

When doing `set_ignored(ignored_flag=False)`, I would suggest `"requested_status"` to be Unresolved

Comment (jjfallete) [reply src/cbapi/response/models.py:299]: Does the ignored flag even do anything? I believe it was added for a future enhancement that hasn't ever been introduced (method was [introduced](https://cbapi.readthedocs.io/en/latest/changelog.html) in cbapi v1.1.1). That said, if an alert is ignored, wouldn't it be ignored from an existing "Unresolved" status?

**Comment by llyon-cb** [comment]
PR has been opened on Jared Fagel's fork to give an opportunity for them to give feedback. Will open a PR to merge directly into master from [my fork](https://github.com/llyon-cb/cbapi-python/tree/response-models) if we don't receive feedback soon-ish.

**Comment by jjfallete** [comment]
> PR has been opened on Jared Fagel's fork to give an opportunity for them to give feedback. Will open a PR to merge directly into master from [my fork](https://github.com/llyon-cb/cbapi-python/tree/response-models) if we don't receive feedback soon-ish.

Thanks @llyon-cb - the opened PR looks good. I'm adding a comment in there.

**Comment by llyon-cb** [comment]
Merged with https://github.com/carbonblack/cbapi-python/pull/279

**Pull request closed without merging**